sympy/simplify/tests/test_simplify.py

```diff
@@ -938,6 +938,11 @@ def test_simplify_kroneckerdelta():
                                      [0, K(0, n), 0, K(1, n)],
                                      [0, 0, K(0, n), 0],
                                      [0, 0, 0, K(0, n)]])
+    assert simplify(eye(1) * KroneckerDelta(0, n) *
```

Comment:
You already showed that these do not fail, so the necessity of the `0*m` has not been secured with a test. I'm guessing you have a push in progress, but just in case...

Reply:
Check two lines down. The oo test fails with returning 0.